code/ecdsap256sha2/Hacl.Impl.ECDSA.fsti

```diff
@@ -119,19 +119,19 @@ val ecdsa_p256_sha2_verify:
     )
 
 (* This code is not side channel resistant *)
-val ecdsa_p256_sha2_verify_u8:
-    mLen: size_t
-  -> m: lbuffer uint8 mLen
-  -> pubKey: lbuffer uint8 (size 64)
-  -> r: lbuffer uint8 (size 32)
-  -> s: lbuffer uint8 (size 32) ->
+val ecdsa_verification_u8:
```

Comment:
Why the name change?
The change breaks things downstream and the rest of the function in this module still use the previous naming scheme.